uni/lib/controller/fetchers/location_fetcher/location_fetcher_asset.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/services.dart' show rootBundle;
+import 'package:redux/redux.dart';
+import 'package:uni/controller/fetchers/location_fetcher/location_fetcher.dart';
+import 'package:uni/model/app_state.dart';
+import 'package:uni/model/entities/location_group.dart';
+
+class LocationFetcherAsset extends LocationFetcher {
+  @override
+  Future<List<LocationGroup>> getLocations(Store<AppState> store) async {
+    final String json =
+        await rootBundle.loadString('assets/text/locations/feup.json');
+    return getFromJSON(json);
+  }
+}
```

Comment:
Why not include getLocations method in LocationFetcher class? I see no need to create an additional class.

Reply:
I would like to keep it, for further use, if the locations are stored in an external API or in the github.

Reply:
Ok.
Because this content is prone to change from time to time, we should consider using an external API